CEF 6533 aarch64_v3 Ships Without Having Been Built

### Operating System Info

Other

### Other OS

Gentoo Linux

### OBS Studio Version

Git

### OBS Studio Version (Other)

_No response_

### OBS Studio Log URL

N/A

### OBS Studio Crash Log URL

_No response_

### Expected Behavior

Building OBS Studio succeeds on an arm64 system with browser enabled.


### Current Behavior

```
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
  Could NOT find CEF (missing: CEF_LIBRARY_WRAPPER_RELEASE) (found suitable
  version "127.0.0", minimum required is "95")

      Reason given by package: Ensure that location of pre-compiled Chromium Embedded Framework is set as CEF_ROOT_DIR.

Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
  cmake/finders/FindCEF.cmake:182 (find_package_handle_standard_args)
  plugins/obs-browser/CMakeLists.txt:12 (find_package)


-- Configuring incomplete, errors occurred!
```


### Steps to Reproduce

1. Build OBS Studio with browser enabled on an arm64 system.


### Anything else we should know?

I haven't been able to locally test this yet myself (while a Raspberry Pi 5 menacingly stares at me from the corner of the room), I've received a Gentoo Linux bug report [1] about this after updating the CEF revision to v3.

Looking into the archive, it seems as if `cmake` has been run, but the build step has not, and the file

`cef_binary_6533_linux_aarch64/build/libcef_dll_wrapper/libcef_dll_wrapper.a`

is missing.

Running the wrapper build step manually before configuring OBS Studio seems to be all that is needed (though again, haven't tested on arm64 yet personally).

Does this seem to be the case, or am I missing something (else)?

Thank you!

1. https://bugs.gentoo.org/955518

--- a/buildspec.json
+++ b/buildspec.json
@@ -33,7 +33,7 @@
                 "macos-x86_64": "d494f1a18746ae65846853c844c1dcf5efa2348e0f422bcbd97059a536f24496",
                 "macos-arm64": "1bb59dbb759150e170796f641a4a84c59c0dea4ffef89477e9d811520af5d15a",
                 "ubuntu-x86_64": "cb7225c7a937ac4cdc9c41700061f45cccc640d696902357782e57f8250bf43a",
-                "ubuntu-aarch64": "f92df7f076bdc8cac2e3c77e27be418008b7168723201cb73fdbc2f6d91bc778",
+                "ubuntu-aarch64": "e82388ef61776a88701b37d96e44f65dcc91f4d4f205f4f17cdc0d50f4e6a7a7",
                 "windows-x64": "922efbda1f2f8be9e5b2754d878a14d90afc81f04e94fc9101a7513e2b5cecc1",
                 "windows-arm64": "df9df4bd85826b4c071c6db404fd59cf93efd9c58ec3ab64e204466ae19bb02a"
             },
@@ -41,7 +41,7 @@
                 "macos-x86_64": 3,
                 "macos-arm64": 3,
                 "ubuntu-x86_64": 3,
-                "ubuntu-aarch64": 3,
+                "ubuntu-aarch64": 4,
                 "windows-x64": 2
             }
         }

